crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -33,7 +20,20 @@ spec:
     status: {}
   version: v1alpha1
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
```

Comment:
Is 
```
    - JSONPath: .status.message
      description: Additional information about workspace state
      name: Info
      type: string
```
is not propagated to v1alpha1.additionalPrinterColumns by purpose? I think we can do it safely since we already have added a message to the status.

Reply:
I think it should be fine to add it, I was mostly just focussed on v1alpha2.